osd_cluster_create/byo-vpc.adoc

```diff
@@ -0,0 +1,35 @@
+[id="byo-vpc"]
+= Managing a Bring-Your-Own Virtual Private Cloud (VPC)
+:context: byo-vpc
+include::modules/attributes-openshift-dedicated.adoc[]
+
+toc::[]
+
+Configuring a cluster to include your own pre-configured Virtual Private Cloud (VPC) allows for the use of any existing network that relies on an existing VPC.
+
+.Prerequisites
+
+* You have a configured VPC. Use the following documentation for guidance on how to configure the VPC's components, such as:
+** a private subnet
+** a public subnet
+** a NAT gateway
+** an Internet gateway
+* You have a configured AWS account in your AWS organization.
+
+.Procedure
+
+. Create an IAM user with the name `osdCcsAdmin`.
+.. On this role, attach the `AdministratorAccess` policy.
+.. Generate an access key pair.
+.. Store this generated key pair for use later in this process.
```

Comment:
instead of "later in this process", maybe specify actually when. I searched for "key" and didn't see it referenced anywhere else in the PR.